aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -199,6 +215,22 @@ void raw_cudnn_convolution_forward_out(
   // TODO: compile empty & fill_ using full_like or full
   Tensor requantize_multiplier_tensor = at::empty(quantized_output.sizes(), at::device(at::kCUDA).dtype(at::kFloat), at::MemoryFormat::ChannelsLast);
   requantize_multiplier_tensor.fill_(requantize_multiplier);
+  c10::optional<at::Tensor> bias_multiplier_tensor;
```

Comment:
yeah I think that would be a good idea. Should we put this on the pytorch doc site and refer them to there or put it in the comments? Might be easier to read math formulas on the site than in the comments

Reply:
since this is an implementation detail I feel it makes more sense to document this in the comments, we can either have inline comments around the operation graph, or for `raw_cudnn_convolution_forward` method (we can move it around later after the refactor to use packed params)